contracts/pox4-pools.clar

```diff
@@ -61,7 +63,7 @@
     (map-set grouped-stackers-len {pool: pool, reward-cycle: reward-cycle} index)))
 
 (define-private (map-set-details (pool principal) (details {lock-amount: uint, stacker: principal, unlock-burn-height: uint, pox-addr: {hashbytes: (buff 32), version: (buff 1)}, cycle: uint}))
-  (let ((reward-cycle (+ (contract-call? 'ST000000000000000000002AMW42H.pox-4 current-pox-reward-cycle) u1))
```

Comment:
why removing this?

Reply:
calling into pox-4 is expensive, therefore, replaced by local function